bot/cogs/manage_classes_cog.py

```diff
@@ -130,7 +130,12 @@ class name as a parameter E.G "cpsc-1010"
         await channel.send(f'Here is your generated class channel {ctx.author.mention}, Good luck!')
 
         #create a class role and mark it as assignable
-        await self.create_role(ctx, class_repr)
+        role = await self.create_role(ctx, class_repr)
+
+        #if cleanup role exists, sync perms
+        if (discord.utils.get(ctx.guild.roles,name="Cleanup")):
```

Comment:
we need to check if it exists and if  it doesnt then create it and mark it as assignable

Reply:
Done